src/components/WebexAvatar/WebexAvatar.js

```diff
@@ -5,16 +5,10 @@ import {Avatar} from '@momentum-ui/react';
 import usePerson from '../hooks/usePerson';
 
 export default function WebexAvatar(props) {
-  let component = null;
-  const person = usePerson(props.personID, props.adapter);
+  const {personID, adapter} = props;
+  const {avatar, displayName, status} = usePerson(personID, adapter);
```

Comment:
Would
`const {avatar, displayName, status} = usePerson({personID, adapter});` work? and we'd get rid of the top definition?

Reply:
no, personID and adapter must be defined in order to be in the right hand side of the `=`